make/hotspot/lib/JvmFeatures.gmk

```diff
@@ -170,12 +170,12 @@ ifeq ($(call check-jvm-feature, link-time-opt), true)
   # later on if desired
   JVM_OPTIMIZATION := HIGHEST_JVM
   ifeq ($(call isCompiler, gcc), true)
-    JVM_CFLAGS_FEATURES += -flto=auto -fuse-linker-plugin -fno-strict-aliasing \
+    JVM_CFLAGS_FEATURES += -DINCLUDE_LINK_TIME_OPTIMIZATION=1 -flto=auto -fuse-linker-plugin -fno-strict-aliasing \
```

Comment:
I don't get this. I find no pre-existing INCLUDE_LINK_TIME_OPTIMIZATION ifdefs, and you are not adding any. What is the purpose of this?

Reply:
I asked the same question.  But as further discussion has indicated, we might need this define for
disabling the poisoning of selected C library functions when building with LTO, because LTO doesn't
support the mechanism used for that.

Reply:
I think all JVM features have these INCLUDE macros so I wanted to it it for this JVM feature too. But I can remove it from the PR if it is just wanted for the other JVM features but not this one for some reason.

Reply:
Yeah, the `link-time-opt` feature is a bit different from the others -- it does not select a different set of sources to be compiled, but affects how the JVM is compiled/linked. I don't think there is a need to define these preprocessor macros just out of pure symmetry reason.

If it turns out in the future that we need to compile code differently, then we can add these defines at that time.